kpops/components/base_components/pipeline_component.py

```diff
@@ -141,11 +141,11 @@ def set_error_topic(self, topic: KafkaTopic) -> None:
         :param topic: Error topic
         """
 
-    def add_extra_output_topic(self, topic: KafkaTopic, role: str) -> None:
+    def add_extra_output_topic(self, topic: KafkaTopic, label: str) -> None:
         """Add an output topic of type extra.
 
         :param topic: Output topic
-        :param role: Role that is unique to the extra output topic
+        :param label: Role that is unique to the extra output topic
```

Comment:
it still says role in the description 🤔

Reply:
fixed! There should be no more `role` around. Appart from the ones that are the values of the test. I don't know if we want to change those